stderr of test 'ID.1209497` in directory 'tests/BugsViaSourgeforce` itself:


# 00:48:17 >  
# 00:48:17 >  Mtimeout -timeout 180 Mserver "--config=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/lib/MonetDB:/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/lib/bin" --set "gdk_dbfarm=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/log" --set mapi_port=32108 --set sql_port=44914 --set xquery_port=51343 --set monet_prompt= --trace "--dbname=mTests_tests_BugsViaSourgeforce" --dbinit="module(pathfinder); pfstart();" ; echo ; echo Over..
# 00:48:17 >  


# 00:48:19 >  
# 00:48:19 >  Mtimeout -timeout 60 MapiClient -lxquery -sxml -u monetdb -P monetdb --host=localhost --port=51343 ID.1209497.xq
# 00:48:19 >  


# 00:48:19 >  
# 00:48:19 >  Done.
# 00:48:19 >  

